connect_four.js

```diff
@@ -1,5 +1,5 @@
-var player1 = 'orange';
-var player2 = 'blue';
+const player1 = 'orange';
+const player2 = 'blue';
 var currentPlayer = player1;
```

Comment:
Can you change `var currentPlayer` to a `let currentPlayer
move let board assignment outside of the finction